.circleci/config.yml

```diff
@@ -207,6 +207,26 @@ jobs:
       - run: git config --global user.email "admin@gatsbyjs.com"
       - run: sh ./scripts/clone-and-validate.sh starters true
 
+  theme_starters_validate:
+    executor: node
+    steps:
+      - checkout
+      - run: ./scripts/assert-changed-files.sh "themes/*|.circleci/*"
+      - run: sh ./scripts/clone-and-validate-theme-starters.sh
+
+  theme_starters_publish:
```

Comment:
Because this is effectively the same as starters_publish, I’d recommend making it a template and exposing variables. (e.g. the way you’ve gone now you _could_ do a separate shell script path, but I’d probably just pass a different glob pattern)